### ISSUE TITLE
кластер bid ask. ошибка?

### ISSUE THREAD

**Issue by Petrov007** [SBProX/SBProX-Dev#1709]
Что сейчас показывает кластер bid ask, похоже на дельту. Но разве он не должен считать объем по бидам и аскам.
![cluster](https://user-images.githubusercontent.com/64303270/100140794-dfdc3200-2eaa-11eb-8bae-d064fe1bc647.png)

**Comment by SBProX** [comment]
Показывает bid-ask. 
Какую дельту? Что такое раздельная?

Формулируйте пожалуйста ваши вопросы понятно, спасибо.

**Comment by SBProX** [comment]
Если вы имеете в виду, почему bid+ask не равно объему, то вероятно из-за того что были сделку внутри спреда, или же за спредом. Но мы это на днях проверим, чтобы удостовериться что нет ошибки

**Comment by Petrov007** [comment]
Да, все верно, имеется ввиду, что кластер объем **не равен** сумме кластер bid + ask.
